test/test_path_utils.rb

```diff
@@ -43,6 +43,12 @@ def test_entries
       "source-maps",
       "symlink"
     ], entries(File.expand_path("../fixtures", __FILE__))
+    
+    [ ['a', 'b'], ['a', 'b', '.', '.'] ].each do |dir_contents|
+      Dir.stub :entries, dir_contents do
+        assert_equal ['a', 'b'], entries(Dir.tmpdir)
```

Comment:
The other assertions in this test don't explicitly reference `Sprockets::PathUtils` either (nor do the assertions in other tests). The structure of the test class make it explicit enough what's being called, I think.